PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -377,6 +378,11 @@ struct HfTaskFlowCharmHadrons {
   // Resolution
   void processResolution(CollsWithQvecs::iterator const& collision)
   {
+
+    if (!collision.sel8() || collision.posZ() > zVtxMax) {
+      return;
```

Comment:
Isn't `std::abs` missing here?

Reply:
sorry, will open a new PR as I have overwritten the previous one

Reply:
You can still reopen this one.